tests/test_cpp_conduit.py

```diff
@@ -2,15 +2,32 @@
 
 from __future__ import annotations
 
-import exo_planet_c_api
-import exo_planet_pybind11
-import home_planet_very_lonely_traveler
+import importlib
+import sys
+
 import pytest
 
 import env
 from pybind11_tests import cpp_conduit as home_planet
 
 
+def import_warns_freethreaded(name):
+    if name not in sys.modules and not getattr(sys, "_is_gil_enabled", lambda: False)():
```

Comment:
```suggestion
    if name not in sys.modules and not getattr(sys, "_is_gil_enabled", lambda: True)():
```

GIL is always enabled unless this is present and False.